ci/build-jenkins.sh

```diff
@@ -27,20 +27,24 @@ ctest --output-on-failure
 
 readonly BTDIR="google/cloud/bigtable"
 
+# Use the APP_PROFILE_ID if set or pass an empty string (represets default profile) otherwaise.
```

Comment:
s/represets/the/
s/otherwaise/otherwise/

Reply:
fixed.